src/browser/modules/EditorInfo/NodeLabel.jsx

```diff
@@ -0,0 +1,24 @@
+/**
+ * This component is used to display the node label on editor drawer
+ */
+import React from 'react'
+import {
+  DrawerSubHeader,
+  DrawerSection,
+  DrawerSectionBody
+} from 'browser-components/drawer'
+import * as _ from 'lodash'
+
+export const NodeLabel = props => {
+  let labels = _.map(props.nodeLabel, label => {
+    return label || null
+  })
+  return labels.length > 0 ? (
+    <DrawerSection>
+      <DrawerSubHeader>Node Label</DrawerSubHeader>
```

Comment:
We would like to show show sub header irrespective of label availability. If there is no label then say so instead of not showing the section entirely

Reply:
done